components/elm/src/data_types/ColumnDataType.F90

```diff
@@ -2966,38 +2957,70 @@ subroutine col_cs_summary(this, bounds, num_soilc, filter_soilc)
        c = filter_soilc(fc)
 
        ! total product carbon
-       this%totprodc(c) = &
+       this%totprodc(c) =      &
             this%prod10c(c)  + &
             this%prod100c(c) + &
             this%prod1c(c) 
 
        ! total ecosystem carbon, including veg but excluding cpool (TOTECOSYSC)
-       this%totecosysc(c) = &
+       this%totecosysc(c) =    &
             this%cwdc(c)     + &
             this%totlitc(c)  + &
             this%totsomc(c)  + &
-            this%totprodc(c) + &
+            this%totprodc(c) + & 
             this%totvegc(c)
 
        ! total column carbon, including veg and cpool (TOTCOLC)
        ! adding col_ctrunc, seedc
-       this%totcolc(c) = &
+
+       this%totcolc(c) =       &
             this%totpftc(c)  + &
             this%cwdc(c)     + &
             this%totlitc(c)  + &
             this%totsomc(c)  + &
             this%totprodc(c) + &
             this%ctrunc(c)   + &
             this%cropseedc_deficit(c)
-            
-       this%totabgc(c) = &
-            this%totpftc(c)  + &
+
+       this%totabgc(c) =       &
             this%totprodc(c) + &
             this%seedc(c)    + &
-            this%ctrunc(c)    
+            this%ctrunc(c)   + & 
+            this%totpftc(c)
+
+
     end do
   end subroutine col_cs_summary 
 
+  ! -----------------------------------------------------------------------
+
+  subroutine col_cs_zero_upscaled_veg(this, bounds, num_soilc, filter_soilc)
+    !
+    ! !DESCRIPTION:
+    ! As an alternative to summarizing vegetation states in CTC and then
+    ! upscaling to the column level, we just zero them when FATES is turned on
+    ! (or other potential models).
+    !
+    ! !ARGUMENTS:
+    class(column_carbon_state) :: this
+    type(bounds_type)      , intent(in)    :: bounds          
+    integer                , intent(in)    :: num_soilc       ! number of soil columns in filter
+    integer                , intent(in)    :: filter_soilc(:) ! filter for soil columns
+    ! locals
+    integer :: fc
+    integer :: c
+
```

Comment:
How about adding `if (.not.use_fates) return` in all `*_zero_upscaled_veg) subroutines?

Reply:
will do